documentation/api/style-guide/index.md

```diff
@@ -0,0 +1,79 @@
+# Style Guide
```

Comment:
As the `style-guide` file grows over time it is probably best we convert this file to use `.mdx` and then create re-usable components to map over colors, h1's and etc...

**For example:**

```JavaScript
let colors = [
  {
    color: 'hmda blue',
    css: '--hmda-blue',
    hex: '#1f498f',
    rgb: '31, 73, 143',
    cmyk: '78, 49, 0, 44'
  },
  {
    color: 'hmda blue light',
    css: '--hmda-blue-light', 
    hex: '#2360C2',
    rgb: '35, 96, 194',
    cmyk: '82, 51, 0, 24'
  },
  {
    color: 'hmda blue dark',
    css: '--hmda-blue-dark',
    hex: '#162e51',
    rgb: '22, 46, 81', 
    cmyk: '73, 43, 0, 68'
  },
  {
    color: 'hmda blue bright',
    css: '--hmda-blue-bright',
    hex: '#3B7BE9',
    rgb: '59, 123, 233',
    cmyk: '75, 47, 0, 9'
  }
]
```

**Then in a React re-usable component called `ColorCard`.**

```JavaScript
{
  colors.map(color => (
    <article key={color.css} className="color-card">
      <div className={`color ${color.css}`}> 
        {color.color}
      </div>

      <p>
        <span>CSS:</span> {color.css} <br/>
        <span>Hex:</span> {color.hex}<br/>
        <span>RGB:</span> {color.rgb}<br/>
        <span>CMYK:</span> {color.cmyk}<br/>
      </p>

    </article>
  )) 
}
```

The same could be done with `h` tags, letter spacing etc...